src/components/NewWorkspaceModal.js

```diff
@@ -367,7 +367,7 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
             h(Link, {
               href: 'https://support.terra.bio/hc/en-us/articles/12029087819291',
               ...Utils.newTabLinkProps
-            }, ['Learn more and follow changes.'])])
+            }, ['Learn more and follow changes', icon('pop-out', { size: 14, style: { marginLeft: '0.25rem' } })])])
```

Comment:
I realize it's only used twice in this PR, and it's a small component, but if this is meant to be used whenever there's an external link, would it be worth it to put it in a common component, with default styling (open question - not meant to be leading)?

Reply:
Agreed, I think it's worth pulling out. I will create a UIE ticket for it.

Reply:
I created https://broadworkbench.atlassian.net/browse/UIE-76 and might do it in this sprint if I end up having time. I would prefer just to not increase the scope if this PR, which already suffered quite a bit of creep.